CHANGELOG.md

```diff
@@ -88,7 +88,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where the ArXiv Fetcher did not support HTTP URLs [koppor#328](https://github.com/koppor/jabref/issues/328)
 - We fixed an issue where only one PDF file could be imported [#4422](https://github.com/JabRef/jabref/issues/4422)
 - We fixed an issue where "Move to group" would always move the first entry in the library and not the selected [#4414](https://github.com/JabRef/jabref/issues/4414)
-
+- We fixed an issue to support sorting using Read Status(Sorting of read-status isn't working as expected #4521)
```

Comment:
I think this bug was introduced in the latest development version and thus no changelog entry is necessary in this case.

Reply:
Done